charts/sps-sample/values.yaml

```diff
@@ -26,7 +26,11 @@ refarch-gateway:
       value: RewritePath=/(?<urlsegments>.*), /$\{urlsegments}
     - name: SPRING_CLOUD_GATEWAY_ROUTES_1_FILTERS_1
       value: RemoveResponseHeader=WWW-Authenticate
-
+  applicationYML:
+      name: SPRING_CLOUD_GATEWAY_ROUTES_1_URI
+      value: "http://{{ include \"refarch-gateway.fullname\" $.Subcharts.haus }}-frontend:8080/"
+      name1: SPRING_CLOUD_GATEWAY_ROUTES_1_PREDICATES_0
+      value1: Path=/**
```

Comment:
natürlich muss das noch in ein spring boot yaml convertiert werden  z. B. mit https://env.simplestep.ca/